### PR TITLE
Pg add weapon class

### PR DESCRIPTION
**Pull request by phillipgray**
Okay, python_thursday folks. Here's my first (of hopefully many more) pull requests. It's a pretty granular change, so hopefully, it'll be a quick add.

Questions about super() have been worked out. Boom.

### CODE REVIEW

File: modules/items.py [hunk 1]
```diff
@@ -6,16 +6,18 @@ class Item():
 
     This is the base class for items
     """
-    def __init__(self, name):
+    def __init__(self, name=None, description=None):
         self.name = name
+        self.description = description
 
-class Bag():
+class Bag(Item):
```
Comment (jamalhansen) [inline modules/items.py]: Curious that you made the bag an item.  I wasn't thinking of it as one, but sure why not :)

File: tests/items_test.py [hunk 2]
```diff
@@ -172,6 +185,18 @@ def test_remove_many_items_will_remove_only_as_many_as_exist(self):
         self.assertEqual(0, self.bag.how_many("rhymes"))
         self.assertEqual("rhymes", item.name)
 
+class WeaponTest(unittest.TestCase):
+    def test_weapon_has_damage_on_init(self):
+        dagger = Weapon(10)
+        self.assertEqual(dagger.damage, 10)
+
+    def test_weapon_has_damage_on_init(self):
```
Comment (jamalhansen) [inline tests/items_test.py]: I think you want this test to have a different name.

**Comment by phillipgray** [comment]
Ahh, missed that one. Fixed now.

I figured, it might be useful at some point to have a player command to look at an item, and see it's name and description. This could always be deprecated later, though.

**Comment by jamalhansen** [comment]
Time to do the rebase dance!

**Comment by jcode89** [comment]
I agree with @jamalhansen It's dancing time, because this pull is looking good :) :+1:

**Comment by phillipgray** [comment]
Woo-hoo! [small rebase dance]
Pruning this branch now.

**Pull request merged**